prow/jobs/jobs.yaml

```diff
@@ -6,4 +6,272 @@ periodics:
     serviceAccountName: pre-submit-service-account
     containers:
     - image: alpine
-      command: ["/bin/date"]
\ No newline at end of file
+      command: ["/bin/date"]
+
+postsubmits:
+  #TODO(vijat@): Find a programmatic way to generate postsubmit jobs for all the service controllers
+  aws-controllers-k8s/sagemaker-controller:
+  - name: sagemaker-post-submit-release
+    decorate: true
+    extra_refs:
+    - org: aws-controllers-k8s
+      repo: test-infra
+      base_ref: main
+      workdir: true
+    spec:
+      serviceAccountName: post-submit-service-account
+      containers:
+        - image: errorbyte/ack-prow-cd #TODO(vijat@): find the correct place for hosting this image.
+          securityContext:
+            privileged: true
```

Comment:
This scares me :/ Is this because of `buildah` failing otherwise?

Reply:
Correct.